### DISCUSSION TITLE
datm evaluations

### DISCUSSION THREAD

**Discussion by wwieder** [NCAR/LMWG_dev#23] [General]
We should start evaluating candidate datm products for land-only simulations with CLM6.  Current options include:
- **GSWP3**       | 1901-2014  | 1.5 hour resolution 
- **CRU-NCEP** | 1901-2015  | 6 hour resolution 
- **CRU-JRA**    | 1901-2022  | 6 hour resolution (used for TRENDY)
- **ERA5**           | 1940-2023 | 

Initially lets run short spinup + historical simulations with each of these in an 1 or 2 degree SP case and then look at ILAMB results. (see #18, #19, #20, #21)

**Comment by slevis-lmwg** [comment]
Discussing with @olyson 
1) gswp3 v1 is default   IN PROGRESS
2) cru-ncep is cru v7   IN PROGRESS
Keith does not recommend cloning because there may be tuning params that go with different datm drivers (eg land tuning mode). So better to use the clm51 compsets.
3) For era5 need to change manually in env_run and keep land tuning mode same as gswp3. Keith proposes cycling the first 10 yrs during spinup, rather than the 20 that we do for the other forcing data. We decided to hold off and do the other three simulations first. We want feedback on spinning up because data starts in 1940.
4) For cru-jra we do not have compset and do not have the option in env_run, so need to point to this manually. Check with Daniel Kennedy who has likely used this for TRENDY and SMILE.

**Reply by slevis-lmwg** [reply]
Also
- run with latest ctsm 5.1 tag
- and 2 degree

**Reply by slevis-lmwg** [reply]
@olyson I decided to go ahead and start the ERA5 cases with your suggestion of cycling the first 10 yrs during spin-up and from 1850 to 1949. Worst case I have to redo this one.

**Comment by olyson** [comment]
ILAMB results for GSWP3V1, CRUNCEPV7, and CRUJRA here:

https://webext.cgd.ucar.edu/I20TR/ctsm51d142_f19_GSWP3V1_hist/lnd/_build_ILAMB2_7_ForcingDatasets/

**Reply by olyson** [reply]
Updated for CRUJRA_flds here:
https://webext.cgd.ucar.edu/I20TR/ctsm51d142_f19_CRUjra_hist_flds/lnd/_build_ILAMB2_7_ForcingDatasets_flds/

**Comment by wwieder** [comment]
CRU-JRA may not be a preferred forcing dataset moving forward 👎 

@olyson noted a big arctic hole in in BGC runs with CRUJRA 
![image](https://github.com/NCAR/LMWG_dev/assets/8031012/34bccdb8-e527-4125-b7c2-b21adabe5ce6)

@dlawrenncar noted lower incoming longwave with CRU-JRA (here for SP runs that @slevis-lmwg did)
![image](https://github.com/NCAR/LMWG_dev/assets/8031012/4d45c927-8aa9-40b9-8700-a9af3f78cb21)

But given previous concerns with incoming longwave, we ran this with model calculated FLDS in #20 and #35

@olyson noted this must be driven by difference in TBOT, QBOT and PBOT.
ILAMB results posted above suggest that arctic temperatures have a [cold bias in the Arctic](https://webext.cgd.ucar.edu/I20TR/ctsm51d142_f19_GSWP3V1_hist/lnd/_build_ILAMB2_7_ForcingDatasets/Forcings/SurfaceAirTemperature/CRU4.02/CRU4.02.html#AllModels)


e.g. here's the seasonal cycle for this region of W. Siberia with CRU-JRA, ironically compared to CRU observations.
![image](https://github.com/NCAR/LMWG_dev/assets/8031012/9329b884-7141-410c-a90b-05c6692ff5fa)

**Comment by olyson** [comment]
A polar plot of boreal summer TLAI from years 61-80 of the CRUJRA_FLDS 1850 pAD simulation (see #40 ).
Some improvement compared to the CLM calculated FLDS simulation (plot above).
![ctsm51_ctsm51d150_ne30pg3ne30pg3mg17_CRUJRA_FLDS_1850pAD JJA TLAI polar](https://github.com/NCAR/LMWG_dev/assets/18671367/30b3f0e2-d867-4a0d-b1ae-11bf6e6f06f3)

**Reply by wwieder** [reply]
Thanks for posting these, Keith.  Toggling back and forth between the noLW and LW simulations  helps to see the regions that are no longer dead. Outside of the high arctic this simulation looks lush!  

I wondered, how iterative do we want to be with additional CRU-JRA experiments (e.g., as above but turn on Sturm)?

**Reply by olyson** [reply]
See below for a polar plot of the differences between this run and our GSWP3 control.  As you mentioned, vegetation is quite a bit healthier at the lower polar latitudes.  Once this is spunup (in progress), this could be considered our control for CRUJRA.  
But I wonder if we should just add in ABCDE (Sturm + VegMods + SNICAR) to get us level with our latest GSWP3 simulation for which we have a full spinup.  I could go either way with the latest SNICAR albedo changes.  Then we can decide what to back off on if necessary perhaps?
![CRUJRA_FLDS-CTSM_GSWP3 JJA TLAI diff polar](https://github.com/NCAR/LMWG_dev/assets/18671367/fb04e972-5afa-49b7-9527-c9fa61243a37)

**Comment by olyson** [comment]
Fyi, see this post from tcraig (Tony Craig I assume) regarding creating an ERA5 forcing dataset.  

https://bb.cgd.ucar.edu/cesm/threads/era5-datm-forcing.9139/

**Comment by slevis-lmwg** [comment]
Thanks @olyson 
I will respond to Tony to let him know that Taydra Low is working on it.

**Reply by slevis-lmwg** [reply]
I private-messaged him for now, rather than sharing the status publicly.

**Comment by wwieder** [comment]
in #54 & #56 @olyson looked at differences in historical simulations initialized with CRU-JRA for 1901-1920 (standard) vs. 1940-1949 climate (as we'd have to do for ERA5).  He concluded that the run initialized with 1940 climate _"Maybe slightly worse for NBP and slightly better for soil carbon.  Everything else looks pretty similar."_. I agree, with the only significant differences over the historical period being in slow pools that have different initial conditions based on the period we're using for initial climate (e.g. [slightly higher total veg C](https://webext.cgd.ucar.edu/I1850/ctsm51_ctsm51d166deadveg_1deg_CRUJRA1940-1949_FLDS_ABsnoCDE_blk_A5BCD_hist/lnd/ctsm51_ctsm51d166deadveg_1deg_CRUJRA1940-1949_FLDS_ABsnoCDE_blk_A5BCD_hist_2003_2022-ctsm51_ctsm51d166deadveg_1deg_CRUJRA_FLDS_ABsnoCDE_blk_A5BCD_hist_2003_2022/set1/set1_TOTVEGC.gif) and [lower soil C](https://webext.cgd.ucar.edu/I1850/ctsm51_ctsm51d166deadveg_1deg_CRUJRA1940-1949_FLDS_ABsnoCDE_blk_A5BCD_hist/lnd/ctsm51_ctsm51d166deadveg_1deg_CRUJRA1940-1949_FLDS_ABsnoCDE_blk_A5BCD_hist_2003_2022-ctsm51_ctsm51d166deadveg_1deg_CRUJRA_FLDS_ABsnoCDE_blk_A5BCD_hist_2003_2022/set1/set1_TOTSOMC.gif), which produces slight differences in total ecosystem C stocks.

![image](https://github.com/NCAR/LMWG_dev/assets/8031012/f30ef136-57de-4e54-b901-85273a26e16c)
